src/lib.rs

```diff
@@ -40,6 +42,6 @@ pub mod synth_minidump;
 pub mod system_info;
 
 pub use iostuff::Readable;
+pub use minidump::*;
```

Comment:
This was re-ordered by rust-fmt which my editor runs automatically on every save.  Would you mind if I made PRs to format the entire project and add CI config to keep running cargo fmt, clippy, tests?

Reply:
That seems fine, yes. I've been historically lazy about ensuring formatting and such on my own personal projects. In my defense this project *does* predate a lot of Rust tooling. :)